docs/compatibility.md

```diff
@@ -15,6 +15,14 @@ problem. Please also look at the current list of
 [bugs](https://github.com/NVIDIA/spark-rapids/issues?q=is%3Aopen+is%3Aissue+label%3Abug) which are
 typically incompatibilities that we have not yet addressed.
 
+## Non-empty nulls
+
+The SQL plugin doesn't support nested types to have non-empty nulls and if the data being processed 
```

Comment:
"non-empty nulls" sounds cryptic. Can you rephrase or illustrate with an example?

Reply:
Yeah, I agree. I can't think of another word, let me explain with an example